Client/Frontend/Settings/HomepageSettings/WallpaperSettings/v2/WallpaperSettingsViewModel.swift

```diff
@@ -0,0 +1,210 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+public enum WallpaperSettingsError: Error {
+    case itemNotFound
+}
+
+class WallpaperSettingsViewModel {
+
+    enum WallpaperSettingsLayout: Equatable {
+        case compact
+        case regular
+
+        // The maximum number of items to display per row
```

Comment:
In the old settings screen, we were showing 6 on landscape and 3 on portrait. Just double checking that we verified that that is changing because I have no context. :)